Base/usr/share/man/man1/test-js.md

```diff
@@ -0,0 +1,37 @@
+## Name
+
+test-js - run the LibJS test suite
+
+## Synopsis
+
+```**sh
+$ test-js [options...]
```

Comment:
Since there's only one option, I'd write it out here instead:
```suggestion
$ test-js [--show-time]
```
But it's also fine this way